docs/source/gbd2017_models/concept_models/vivarium_conceptmodel_template/concept_model.rst

```diff
@@ -31,11 +45,13 @@
   And then add it to the list of decorators above.
 
 
-.. _concept_model_template:
 
-======================
-Concept model template
-======================
+
+.. _{YOUR_MODEL_UNDERSCORE}:
+
+==================
+{YOUR_MODEL_TITLE}
+==================
```

Comment:
Will this make the "Concept model template" page show up as "{YOUR_MODEL_TITLE}" in the index?

Also, does this page even show up in the index?